js/src/tab.js

```diff
@@ -43,9 +43,8 @@ const CLASS_NAME_FADE = 'fade'
 const CLASS_NAME_SHOW = 'show'
 
 const SELECTOR_DROPDOWN = '.dropdown'
-const SELECTOR_NAV_LIST_GROUP = '.nav, .list-group'
+const SELECTOR_NAV_PARENT = '.nav, .list-group, .btn-toolbar'
 const SELECTOR_ACTIVE = '.active'
-const SELECTOR_ACTIVE_UL = ':scope > li > .active'
 const SELECTOR_DATA_TOGGLE = '[data-toggle="tab"], [data-toggle="pill"], [data-toggle="list"]'
```

Comment:
I think there may be some unique selector here instead of these three 🤔
Ref: #31976
@XhmikosR @Johann-S